dedibox-network/rpn/quickstart.mdx

```diff
@@ -10,9 +10,9 @@ dates:
   posted: 2021-05-26
 ---
 
-The RPN (Real Private Network) is a dedicated private network connected to the second network interface of your server.  It allows you to exchange data between your servers over a private network that you control.
-This network provides high performance with very low latency and support of jumbo frames as standard. You can easily regroup your servers within your private network with a few clicks, regardless of their number or physical location. RPN v2 gives you the flexibility to build your architecture in a secure and isolated environment. You control everything and can setup your own VLANs, IP addresses ranges, subnets, routing tables and network gateways.
-RPN v2 also allows you to build security layers to manage your private network. Create a public group for your web server applications and place your backend applications in a private group without internet access. All that backed by an insane connectivity for extreme network performances.
+The RPN (Real Private Network) is a dedicated Private Network connected to the second network interface of your server.  It allows you to exchange data between your servers over a Private Network that you control.
```

Comment:
I think here it isn't a product name (only for Elements)

Reply:
```suggestion
The RPN (Real Private Network) is a dedicated private network connected to the second network interface of your server.  It allows you to exchange data between your servers over a private network that you control.
```